nexus/src/external_api/http_entrypoints.rs

```diff
@@ -2609,7 +2609,7 @@ async fn networking_loopback_address_delete(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
-/// Get loopback addresses, optionally filtering by id
+/// List loopback addresses
```

Comment:
Didn't think "optionally filtering by id" is necessary, also not sure it's even true